pages/login.tsx

```diff
@@ -6,12 +6,22 @@ import {
     Input,
     FormControl,
 } from "@chakra-ui/react";
+import { getSession, GetSessionOptions, signIn } from "next-auth/client";
+import { useState } from "react";
 
 /**
  * This is the page that a user will use to either login or register
  * to the SDC platform
  */
 export default function Login(): JSX.Element {
+    // hook to hold the user's email
+    const [email, setEmail] = useState("");
+
+    // signInWithEmail sends a login request to the user's email
+    const signInWithEmail = () => {
+        signIn("email", { email: email });
```

Comment:
Let's make `email` a const?

Reply:
Also, I think it can just be:
```suggestion
        signIn("email", { email });
```
Let me know if this works.